benchmark/GraphServiceClientBenchmark.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Identity.Web;
+using Microsoft.Graph;
+using Microsoft.Extensions.Configuration;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+
+#pragma warning disable CA1050 // Declare types in namespaces
```

Comment:
Why not declare in the same ns as Program.cs ?daemon_console

Reply:
reverted